azure-client-runtime/src/main/java/com/microsoft/azure/v2/AzureTokenCredentialsHandler.java

```diff
@@ -0,0 +1,26 @@
+package com.microsoft.azure.v2;
+
+import com.microsoft.azure.credentials.AzureTokenCredentials;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelOutboundHandlerAdapter;
+import io.netty.channel.ChannelPromise;
+import io.netty.handler.codec.http.HttpRequest;
+
+public class AzureTokenCredentialsHandler extends ChannelOutboundHandlerAdapter {
```

Comment:
This appears to be a Netty specific handler. If I have classes like that, I usually add the technology that they are specific to either in their package or in their class name, especially since ChannelOutboundHandlerAdapter isn't clear that its a Netty class.